packages/samples/headless-commerce-react/src/components/standalone-search-box/standalone-search-box.tsx

```diff
@@ -25,12 +30,17 @@ export default function StandaloneSearchBox(props: IStandaloneSearchBoxProps) {
     }
   }, [state.redirectTo, navigate, state.value, controller]);
 
+  const onSearchBoxInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    controller.updateText(e.target.value);
+    instantProductsController.updateQuery(e.target.value);
+  };
+
   return (
     <div className="StandaloneSearchbox">
       <input
         className="SearchBoxInput"
         value={state.value}
-        onChange={(e) => controller.updateText(e.target.value)}
+        onChange={(e) => onSearchBoxInputChange(e)}
```

Comment:
```suggestion
        onChange={onSearchBoxInputChange}
```

Reply:
I think this works here, but be careful with [`this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this)